extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -293,7 +297,7 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 	}
 
 	@debounce(1500)
-	async fireBookRefresh(book: BookModel): Promise<void> {
+	async initializeBookContents(book: BookModel): Promise<void> {
```

Comment:
This doesn't solve the problem I brought up though - debounce doesn't care about different parameters. Are you planning on addressing that separately?

Reply:
I'll address the debounce issue and making the setFileWatcher return a disposable on a different pr.
https://github.com/microsoft/azuredatastudio/pull/14177#discussion_r571322391